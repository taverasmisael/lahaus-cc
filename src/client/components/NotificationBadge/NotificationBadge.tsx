import classNames from 'classnames'

function NotificationBadge({ className }: { className?: string }) {
  return (
    <span
      className={classNames(
        'rounded-full w-3 h-3 absolute top-0 right-0 bg-accent',
        className
      )}
    />
  )
}

export default NotificationBadge
