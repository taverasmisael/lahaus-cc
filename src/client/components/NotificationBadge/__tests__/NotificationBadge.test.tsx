import { render } from '@testing-library/react'

import NotificationBadge from '../NotificationBadge'

describe('components/NotificationBadge', () => {
  test('it renders correctly', () => {
    const { container } = render(<NotificationBadge />)
    expect(container).toMatchSnapshot()
  })
  test('it applies className correctly', () => {
    const { container } = render(<NotificationBadge className="bg-red-300" />)
    expect(container).toMatchSnapshot()
  })
})
