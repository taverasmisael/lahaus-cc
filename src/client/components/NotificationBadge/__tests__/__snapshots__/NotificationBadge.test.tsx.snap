// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/NotificationBadge it applies className correctly 1`] = `
<div>
  <span
    class="rounded-full w-3 h-3 absolute top-0 right-0 bg-accent bg-red-300"
    data-testid="notification-badge"
  />
</div>
`;

exports[`components/NotificationBadge it renders correctly 1`] = `
<div>
  <span
    class="rounded-full w-3 h-3 absolute top-0 right-0 bg-accent"
    data-testid="notification-badge"
  />
</div>
`;
